README.md

```diff
@@ -1 +1,19 @@
 # Scry
+
+A small tool to analyze the logs of a failed opam build and to give the hint about the cause at the best effort.
+
+#####To Build:
+```
+git clone git@github.com:ocamllabs/ocaml-scry.git
```

Comment:
can you replace this by `opam pin add scry  https://github.com/ocamllabs/ocaml-scry.git`?